package.json

```diff
@@ -125,27 +131,25 @@
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
     "rimraf": "^3.0.2",
-    "storybook": "^7.0.23",
+    "storybook": "^8.2.4",
     "tsup": "^6.6.3",
     "typescript": "^4.9.5",
-    "vite": "^4.1.4",
-    "vue": "^3.3.0",
+    "vite": "^4.5.3",
+    "vue": "^3.4.31",
     "vuestic-ui": "^1.6.6",
     "zx": "^1.14.1"
   },
   "peerDependencies": {
-    "@storybook/blocks": "^7.0.0",
-    "@storybook/builder-vite": "^7.0.0",
-    "@storybook/components": "^7.0.0",
-    "@storybook/core-events": "^7.0.0",
-    "@storybook/manager-api": "^7.0.0",
+    "@storybook/blocks": "^8.2.4",
+    "@storybook/builder-vite": "^8.2.4",
+    "@storybook/components": "^8.2.4",
+    "@storybook/core-events": "^8.2.4",
+    "@storybook/manager-api": "^8.2.4",
     "@storybook/mdx2-csf": "^1.1.0",
-    "@storybook/preview-api": "^7.0.0",
-    "@storybook/theming": "^7.0.0",
-    "@storybook/types": "^7.0.0",
-    "react": "^16.8.0 || ^17.0.0 || ^18.0.0",
```

Comment:
[React peer dependency is no longer required](https://storybook.js.org/docs/addons/addon-migration-guide#react-peer-dependency-is-no-longer-required)